src/Buildalyzer/AnalyzerResult.cs

```diff
@@ -100,7 +100,7 @@ internal AnalyzerResult(string projectFilePath, AnalyzerManager manager, Project
         public IEnumerable<string> ProjectReferences =>
             Items.TryGetValue("ProjectReference", out IProjectItem[] items)
                 ? items.Select(x => AnalyzerManager.NormalizePath(
-                    Path.Combine(Path.GetDirectoryName(ProjectFilePath), x.ItemSpec)))
+                    Path.Combine(Path.GetDirectoryName(ProjectFilePath), x.ItemSpec))).Distinct()

```

Comment:
Should case sensitivity be taken into account here?

Reply:
Thanks for catching that, should be resolved with second commit.
Also added test case for this change.